packages/keystone/asyncLocalStorage.js

```diff
@@ -0,0 +1,44 @@
+const { AsyncLocalStorage } = require('node:async_hooks')
+
+const { getLogger } = require('./logging')
+
+const ASYNC_LOCAL_STORAGES = {}
+const logger = getLogger('asyncLocalStorage')
+
+/**
+ * Return current execution context
+ */
+function getExecutionContext () {
```

Comment:
why do we need more than one context? it's easy to get executionContext as an object.
```
const ctx = getExecutionContext ()
console.log(ctx.reqId, ctx.taskId, ctx.execId) // one should be not null!
```

Reply:
```

if (ctx.reqId) {
  console.log(`we are in HTTP context! user =`, ctx.userId, `fingerprint =`, ctx.fingerprint, ctx.startedAt)
} else if (ctx.taskId) {
  console.log(`we are in worker task! scheduled by processing req =`, ctx.byReqId, ctx.startedAt)
} else {
  console.log(`we are in some script`, ctx.execArgv, ctx.startedAt)
}
```